src/megachatapi_impl.cpp

```diff
@@ -4551,8 +4551,11 @@ void MegaChatApiImpl::onInitStateChange(int newState)
 
 void MegaChatApiImpl::onChatNotification(karere::Id chatid, const Message &msg, Message::Status status, Idx idx)
 {
-    if (megaApi->isChatNotifiable(chatid))
-     {
+    if (megaApi->isChatNotifiable(chatid)
```

Comment:
If we add this check here instead of `ChatRoom::onMessageStatusChange`, I think we miss the other usages of `onChatNotification`:
 - `ChatRoom::onRecvNewMessage(chatd::Idx idx, chatd::Message& msg, chatd::Message::Status status)`
 - `void ChatRoom::onMessageEdited(const chatd::Message& msg, chatd::Idx idx)`
In both methods, we check `status == chatd::Message::kNotSeen` to call `parent.mKarereClient.app.onChatNotification(mChatid, msg, status, idx)` then for this other cases we miss the call to `fireOnChatNotification(chatid, message);`

Reply:
Good catch!! Thanks, Alber. Please, see my next commit: `15dd68bf4ddc2d178790b3e4af827bdf886f6bc6`